src/test/java/org/folio/inventory/dataimport/consumers/DataImportKafkaHandlerTest.java

```diff
@@ -73,7 +73,7 @@
   private JobProfile jobProfile = new JobProfile()
     .withId(UUID.randomUUID().toString())
     .withName("Create instance")
-    .withDataType(org.folio.JobProfile.DataType.MARC);
+    .withDataType(org.folio.JobProfile.DataType.MARC_BIB);
```

Comment:
Does this mean that any existing job profiles with a data type of `MARC` need to be changed to `MARC_BIB`?

Reply:
Yes. we shouldn't have record type `MARC`, for record type we have - `MARC_BIB`, `MARC_AUTHORITY`, `MARC_HOLDING`.
For source we have `MARC`, but in this case `MARC` is a format

Reply:
Does that mean there is going to be a upgrade process to move the existing job profiles to the new data type?

Reply:
Existing job profiles will be migrated in the another module - mod-data-import-converter-storage. You can find script for migration here: https://github.com/folio-org/mod-data-import-converter-storage/pull/164/files#diff-dcf9376ead7f99847995503e9dd1b771daf05bc357b1c117c2655e06b1320edb

Or maybe you talking about `mod_inventory_storage`.  But I double-check and didn't find dataType in the `instance`, `item` or holding. Only field `source`.

Reply:
> Existing job profiles will be migrated in the another module - mod-data-import-converter-storage

Thanks for clarifying that

> Or maybe you talking about mod_inventory_storage

I wasn't, only the owner of the job profiles